src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientStorageAccessor.java

```diff
@@ -63,7 +63,7 @@ public BtcReleaseClientStorageAccessor(
                 throw new InvalidStorageFileException(message, e);
             }
         }
-        if (!readResult.getSuccess()) {
+        if (Boolean.FALSE.equals(readResult.getSuccess())) {
```

Comment:
why this change? i think we are not doing that anywhere else

Reply:
To solve a Sonar complaint.

![image](https://github.com/rsksmart/powpeg-node/assets/94651730/ff3c2a0c-8391-47b9-8d3c-df688d233962)